### PR TITLE
Add flag to update finished URL 

### PR DESCRIPTION
**Pull request by Lyxnx**
Just makes it easier to edit the finished URL from command line rather than having to manually do it

### CODE REVIEW

**Comment by aurieh** [comment]
@Lyxnx could you rebase last 2 commits?

File: script.sh [hunk 1]
```diff
@@ -199,7 +199,7 @@ function upload() {
 		d=$2
 		if [ "$d" = true ]; then
 				clipboard "https://$output_url/$item"
-			echo "https://$output_url/$item"
+			echo "https://$output_url/$item"http://repo.dmulloy2.net/nexus/repository/public/
```
Comment (aurieh) [inline script.sh]: 🤔

File: script.sh [hunk 2]
```diff
@@ -216,7 +216,7 @@ function upload() {
 }
 
 function runupdate() {
-	cp $owodir/conf.cfg $owodir/conf_backup_$current_version.cfg
+    cp $owodir/conf.cfg $owodir/conf_backup_$current_version.cfg
```
Comment (deansheather) [inline script.sh:219]: ?

**Comment by aurieh** [comment]
Actually, rebase all of them

File: script.sh (hunk 1 above)
Comment (Lyxnx) [reply script.sh]: was in clipboard for some reason

File: script.sh (hunk 2 above)
Comment (Lyxnx) [reply script.sh:219]: didn't actually edit that line but git shows I did - no idea why

**Comment by Lyxnx** [comment]
@aurieh never actually come across the rebase command so I don't really know how it works tbh

**Comment by aurieh** [comment]
ddg `git rebase -i`

**Comment by Lyxnx** [comment]
Still have no idea how that works :/

**Comment by islacantwin** [comment]
Wat

**Comment by islacantwin** [comment]
Also this links to a weird domain.

**Comment by islacantwin** [comment]
Give explanation or closing PR.

**Comment by aurieh** [comment]
@OkashiOdayakana it was explained on my comment on the first commit

**Comment by islacantwin** [comment]
O nvm

**Comment by islacantwin** [comment]
Also it's not even taking input for the -fu. Closing PR

**Pull request closed without merging**